docs/pages/bare/using-expo-client.md

```diff
@@ -11,7 +11,7 @@ Inside a freshly initialized bare project, run `npx expo start`, and you can now
 
 ## What are the limitations?
 
-You will not be able to use the parts of your app that require custom native code. To run your bare app in Expo Go, you need to avoid calling any custom native code (native code that isn't included in the Expo SDK). For some apps this may mean that you won't be able to use the Expo Go app almost at all &mdash; for example, if your app depends on custom native code for something as fundamental as navigation or state management (eg: Realm or the Firebase native SDK) then not much of your app will be usable in the client. If your app only has some in app purchases, analytics, a custom map view, an AR view, and so on, then this may actually work great for you &mdash; that particular functionality would not be usable in Expo Go but the rest of the app still would be.
+You will not be able to use the parts of your app that require custom native code. To run your app in Expo Go, you need to avoid calling any custom native code that isn't included in the Expo Go app. For some apps this may mean that you won't be able to use the Expo Go app almost at all &mdash; for example, if your app depends on custom native code for something as fundamental as navigation or state management (eg: Realm or the [Firebase native SDK](/guides/setup-native-firebase)) then not much of your app will be usable in the Expo Go app. If your app has features that are easy to mock like in-app purchases, analytics, a custom map view, and so on, then this may actually work great for you &mdash; that particular functionality would not be usable in Expo Go but the rest of the app still would be.
 
 ## Why might you want to do this?
```

Comment:
we might want to update this section, basically the only benefit is easy previewing of a project with an app that's already downloadable from the store. we can push ppl to dev client instead for all other reasons

Reply:
opened a linear task https://linear.app/expo/issue/ENG-6216/update-mapview-android-standalone-docs

Reply:
Moved to https://github.com/expo/expo/pull/18978